WordPress/src/main/res/values/strings.xml

```diff
@@ -935,6 +935,7 @@
     <string name="error_refresh_unauthorized_comments">You don\'t have permission to view or edit comments</string>
     <string name="error_refresh_unauthorized_pages">You don\'t have permission to view or edit pages</string>
     <string name="error_refresh_unauthorized_posts">You don\'t have permission to view or edit posts</string>
+    <string name="error_refresh_profile_not_accessible">There was an error trying to access your profile information</string>
```

Comment:
As with my earlier comment, can this be shortened? "Unable to access your profile" works.